src/DynamoCore/Models/WorkspaceModel.cs

```diff
@@ -384,7 +384,7 @@ public bool HasUnsavedChanges
             } 
         }
 
-        public GraphLayout.Graph LayoutGraph;
+        public List<GraphLayout.Graph> LayoutSubgraphs;
```

Comment:
Unfortunately `IEnumerable` does not support `.Add()` function and this will cause more changes.